Create an Output action (#108)
hashicorp/terraform-github-actions: examples/outputs.md

> # Terraform Outputs

Reviewer: Singular over plural.
```suggestion
# Terraform Output
```

> If you need the outputs of your terraform later in your Github Actions workflow, you can use the outputs command.

Reviewer: Some grammar fixes.
```suggestion
If you need the outputs of your Terraform configuration later in your GitHub Actions workflow, you can use the `output` subcommand.
```

> You can then use `jq` for example to parse out values.

Reviewer: I think it's better to keep this example about the functionality of the new feature, rather than about `jq`.
```suggestion
```

> In this example the tf_outputs look like:

Reviewer: Using suggested prefix.
```suggestion
In this example, `tf_actions_output` would contain the following.
```

```json
{
  "database": {
    "value": {
      "name": "test-database"
    }
  }
}

> ```

Reviewer: Let's move this after the example YAML.

```yaml
name: 'Terraform GitHub Actions'
on:

>   - push

Reviewer: This is for consistency in the examples.
```suggestion
  - pull_request
```

jobs:
  terraform:
    name: 'Terraform'
    runs-on: ubuntu-latest
    steps:
      - name: 'Checkout'
        uses: actions/checkout@master

> 

Reviewer: ```suggestion
```

Reviewer: Let's get rid off the empty lines in between YAML list items.

>       - name: 'Terraform Outputs'

Reviewer: Singular over plural.
```suggestion
      - name: 'Terraform Output'
```

        id: terraform
        uses: hashicorp/terraform-github-actions@master
        with:
          tf_actions_version: 0.12.13

>           tf_actions_subcommand: 'outputs'

Reviewer: Subcommand is `output`, not `outputs`.
```suggestion
          tf_actions_subcommand: 'output'
```

          tf_actions_working_dir: '.'

> 

Reviewer: Let's get rid off the empty lines in between YAML list items.
```suggestion
```

>       - name: 'Use Terraform Outputs'

Reviewer: Singular over plural.
```suggestion
      - name: 'Use Terraform Output'
```

>         run: echo ${{ steps.terraform.outputs.tf_outputs }}

Reviewer: Using suggested prefix.
```suggestion
        run: echo ${{ steps.terraform.outputs.tf_actions_output }}
```

> 

Reviewer: Let's get rid off the empty lines in between YAML list items.
```suggestion
```

      - name: 'Pull specific database name from outputs'
        run: |
          # Install jq in this shell
          apt-get install jq

          # Parse the outputs from the 'Terraform Outputs' step and grab the database name
          DBNAME=$(echo ${{ steps.terraform.outputs.tf_outputs }} | jq -r '.database.value.name')

          # Will echo out 'test-database'

>           echo $DBNAME

Reviewer: I'm on the fence about leaving this in here. It's more about using `jq` than the functionality of `tf_actions_output`.
